### PR TITLE
Usage clarification of the group-timeout property.

### PR DESCRIPTION
**Pull request by warbabun**
Added some clarification on the usage of  the group-timeout property based on discussion here https://stackoverflow.com/questions/48230453/issue-with-spring-integration-aggregator-group-timeout-value

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@warbabun Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-integration&pullRequestId=3019)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-integration&pullRequestId=3019) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@warbabun Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-integration&pullRequestId=3019)!

File: src/reference/asciidoc/aggregator.adoc [hunk 1]
```diff
@@ -470,7 +470,7 @@ By default, an internal `DefaultLockRegistry` is used.
 Use of a distributed `LockRegistry`, such as the `ZookeeperLockRegistry`, ensures only one instance of the aggregator can operate on a group concurrently.
 See <<./redis.adoc#redis-lock-registry,Redis Lock Registry>>, <<./gemfire.adoc#gemfire-lock-registry,Gemfire Lock Registry>>, and <<./zookeeper.adoc#zk-lock-registry,Zookeeper Lock Registry>> for more information.
 <21> A timeout (in milliseconds) to force the `MessageGroup` complete when the `ReleaseStrategy` does not release the group when the current message arrives.
-This attribute provides a built-in time-based release strategy for the aggregator when there is a need to emit a partial result (or discard the group) if a new message does not arrive for the `MessageGroup` within the timeout.
+This attribute provides a built-in time-based release strategy for the aggregator when there is a need to emit a partial result (or discard the group) if a new message does not arrive for the `MessageGroup` within the timeout which counts from the time the last message arrived. To set up a timeout which counts from the time the `MessageGroup` was created see `group-timeout-expression` information.
```
Comment (artembilan) [inline src/reference/asciidoc/aggregator.adoc]: Please, obey the rule *One sentence per line*: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Thanks

**Review by artembilan — CHANGES_REQUESTED** [review]
Just one formatting concern.

**Comment by artembilan** [comment]
Gentle ping. We can fix formatting issues on merge, but would be great to know if you agree with that or you are still going to fix it yourself.

Thanks

**Comment by warbabun** [comment]
Done @artembilan

**Comment by artembilan** [comment]
@warbabun ,

Thank you for the contribution; looking forward for more!

**Comment by warbabun** [comment]
Same here!

**Pull request merged**